ceaos/objects/beds.py

```diff
@@ -24,6 +24,18 @@ def delete_plant(self, plant):
         del self.plants[plant.name]
         self.num_plants -= 1
 
+    def recv_details(self, **kwargs):
```

Comment:
I think I'm unclear as to the intent of this function. Seems like the data must already be received for this function to work?